### PR TITLE
C_CPP : Snake and Ladder Problem Using graph

### PR DESCRIPTION
**Pull request by Pranshu321**
<hr>

## Description 📜

In this, I have explained how we can solve snake and ladder problems using the graph with the most optimized approach with easy-to-understand code and useful comments.
Issue Number Closed: #8382 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)
 

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Data Structures/Graph/Snake and Ladder Problem/Snake_and_Ladder.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Output :
+```
+Number of minimum moves : 3
+```
+### Time Complexity
+*O(n)* here n is the nodes of the graphs.
+
+### Space COmplexity
+*O(n)* here n is the nodes of the graphs.
+
```
Comment (vasu-1) [inline C_CPP/Data Structures/Graph/Snake and Ladder Problem/Snake_and_Ladder.md:100]:
```suggestion
### Space Complexity
*O(n)* here n is the nodes of the graphs.
```

**Review by vasu-1 — CHANGES_REQUESTED** [review]

File: C_CPP/Data Structures/Graph/Snake and Ladder Problem/Snake_and_Ladder.md (hunk 1 above)
Comment (Pranshu321) [reply C_CPP/Data Structures/Graph/Snake and Ladder Problem/Snake_and_Ladder.md]: Changes Done please review it again @vasu-1.

**Review by vasu-1 — APPROVED** [review]
LGTM 🌝 @Pranshu321 

@Suvraneel @Saviour1001

**Comment by Suvraneel** [comment]
@Pranshu321  LGTM ❤️  🚀  Awesome work Pranshu

And thanks for being one of the most active contributors of this domain 🖤

**Pull request merged**